testing/trino-testing/src/main/java/io/trino/testing/BaseOrcWithBloomFiltersTest.java

```diff
@@ -11,41 +11,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.plugin.hive.orc;
+package io.trino.testing;
 
 import io.trino.Session;
-import io.trino.plugin.hive.HiveQueryRunner;
-import io.trino.testing.AbstractTestQueryFramework;
-import io.trino.testing.QueryRunner;
 import org.intellij.lang.annotations.Language;
 import org.testng.annotations.Test;
 
 import static io.trino.testing.sql.TestTable.randomTableSuffix;
 import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class TestOrcWithBloomFilters
+public abstract class BaseOrcWithBloomFiltersTest
         extends AbstractTestQueryFramework
 {
-    @Override
-    protected QueryRunner createQueryRunner()
-            throws Exception
-    {
-        return HiveQueryRunner.builder()
-                .addHiveProperty("hive.orc.bloom-filters.enabled", "true")
-                .addHiveProperty("hive.orc.default-bloom-filter-fpp", "0.001")
-                .build();
-    }
+    protected abstract String getTableProperties(String bloomFilterColumnName, String bucketingColumnName);
 
     @Test
     public void testOrcBloomFilterIsWrittenDuringCreate()
     {
         String tableName = "create_orc_with_bloom_filters_" + randomTableSuffix();
         assertUpdate(
                 format(
-                        "CREATE TABLE %s WITH (%s) AS SELECT orderstatus, totalprice FROM tpch.tiny.orders",
+                        "CREATE TABLE %s WITH (%s) AS SELECT orderstatus, totalprice FROM tpch.tiny.orders ORDER BY totalprice",
```

Comment:
ORDER BY here is redundant in this query, so don't use it

Reply:
I've tested it before. Whether it's sorted or not doesn't affect the test result. So I uniformly remove order by. Hive related changes are an independent submission in itself.